pkg/analytics/analytics.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2019 LitmusChaos Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package analytics
+
+import (
+	"crypto/rand"
+	"fmt"
+
+	ga "github.com/jpillora/go-ogle-analytics"
+)
+
+const (
+	// ClientID contains TrackingID of the application
+	ClientID = "UA-92076314-21"
+
+	// supported event categories
+
+	// Category category notifies installation of a component of Litmus Infrastructure
+	Category = "Litmus-Infra"
+
+	// supported event actions
+
+	// Action is sent when the installation is triggered
+	Action = "Installation"
+
+	// supported event labels
+
+	// Label denotes event is associated to which Litmus component
+	Label = "Chaos-Operator"
```

Comment:
Make all variable in camelcase not big camelcase, Like: `ClientID` -> `clientID`

Reply:
okay done!